config/foreman-proxy-content.migrations/200323151921-reset_tuning.rb

```diff
@@ -0,0 +1,2 @@
+scenario[:facts] = {} unless scenario.key?(:facts)
```

Comment:
A common short hand for this in Ruby is
```suggestion
scenario[:facts] ||= {}
```

However, due to https://github.com/theforeman/foreman-installer/blob/develop/config/foreman-proxy-content.migrations/190904144224-set_tuning_fact.rb it's also safe to leave it out.

```suggestion
```

Reply:
I wanted to keep this specific migration safe and not dependent on the previous tuning migration. If for some reason, an advanced user hand edits this configuration, it is probably safe to check and initialize this to { } before acting on it.

Reply:
I'm not opposed to defensive coding and in fact think too often we're not defensive enough so I'm fine with it here.